cloudant-spark-sql/build.sbt

```diff
@@ -1,6 +1,6 @@
 organization  := "com.cloudant.spark"
 
-name := "spark-sql"
+name := "cloudant-spark"
 
```

Comment:
name has to be `spark-cloudant` to comply with our package http://spark-packages.org/package/cloudant-labs/spark-cloudant